domains/Juandafp.is-a.dev.json

```
@@ -0,0 +1,8 @@
+{
+  "owner": {
+    "email": "tucorreo@ejemplo.com"
```

username is required```suggestion
    "username": "J-D-FARFAN",
    "email": "tucorreo@ejemplo.com"
```